.bumpversion.cfg

```diff
@@ -0,0 +1,7 @@
+[bumpversion]
+current_version = 0.1.4
+commit = True
+
```

Comment:
Intentionally dont tag since we will do this via https://github.com/loicleyendecker/deeppath/releases/new